src/orange/orangeinp/Solid.hh

```diff
@@ -21,12 +21,12 @@ namespace orangeinp
 {
 //---------------------------------------------------------------------------//
 /*!
- * Define the angular region of a solid.
+ * Define the azimuthal truncation of a solid.
  *
  * This is a pie slice infinite along the z axis and outward from it. Its cross
  * section is in the \em x-y plane, and a start
  * angle of zero corresponding to the \em +x axis. An interior angle of one
- * results in no radial excluded in the resulting solid. A interior angle of
+ * results in no radial exclusion from the resulting solid. A interior angle of
  * more than 0.5 turns (180 degrees) results in a wedge being subtracted from
  * the solid, and an angle of less than or equal to 0.5 turns results in the
  * intersection of the solid with a wedge.
```

Comment:
Not in this MR, but I think in line 36:

```
EnclosedAzi{Turn{-0.125}, Turn{0.25}};
```

The starting point should be -0.25

Reply:
Actually no , starting at -0.25 and an interior of 0.25 would be the bottom right (southeast) quadrant; this means from SE to NE.